website/.sailsrc

```diff
@@ -5,5 +5,281 @@
   "_generatedWith": {
     "sails": "1.2.5",
     "sails-generate": "2.0.0"
+  },
+  "builtFromMarkdown": {
```

Comment:
Let's omit this change (it's something I would put into the gitignore in a separate config file except that I wanted to try and keep it all in one place)

This change happened automatically because of the script that compiles the HTML -- so you can just change it back to what it was before and will be good to go